apps/teamsjs-diagnostic-app/src/components/sample/LogConsole.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import './LogConsole.css';
+
+interface LogConsoleProps {
+  initialLogs?: string[];
+}
+
+const MAX_LOGS = 100;
```

Comment:
Why are we going to capture only 100 logs? And how did you come up with this number? Also, what if the application errors out on 101?

Reply:
Can you tell me how are you going to resolve it in MVP?